Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -195,7 +201,9 @@ private void CreateRange()
             else
             {
                 this.lower = limits[0];
-                this.upper = limits[1];
+                this.upper = limits[1] > Int32.MaxValue
+                    ? limits[1]
+                    : limits[1] + 1;

```

Comment:
I'm probably missing something obvious, but I can't seem to understand what's the purpose of this change. :confused: